src/transformers/models/vit_mae/__init__.py

```diff
@@ -33,6 +33,13 @@
         "ViTMAEPreTrainedModel",
     ]
 
```

Comment:
The import needs to be `from ...file_utils import _LazyModule, is_tf_available, is_torch_available, is_vision_available` if not already done.